crossy-web/app/(app)/play/games/[slug]/gameboard.tsx

```diff
@@ -37,7 +37,7 @@ type Props = {
   setClueNum: React.Dispatch<React.SetStateAction<number>>
   highlights?: Record<number, string>
   setHighlights?: React.Dispatch<React.SetStateAction<Record<number, string>>>
-  friendsLocations?: Record<string, number>
+  friendsLocations?: Record<string, { location: number; direction: string }>
```

Comment:
todo (self): `type Direction = "across" | "down"` should probably be an exported type